Src/WindowsServer/WindowsServer.Shared.Tests/WindowsServer.Shared.Tests.projitems

```diff
@@ -9,6 +9,7 @@
     <Import_RootNamespace>Microsoft.ApplicationInsights.WindowsServer</Import_RootNamespace>
   </PropertyGroup>
   <ItemGroup>
+    <Compile Include="$(MSBuildThisFileDirectory)AppServicesHeartbeatTests.cs" />
```

Comment:
AppServicesHeartbeatTelemetryModuleTests might be better name.

Reply:
Ok.